resources/sushy-tools/Dockerfile

```diff
@@ -1,7 +1,15 @@
 FROM registry.hub.docker.com/library/python:3.9
 
+ENV SUSHY_TOOLS_VERSION="0.18.1"
+
 RUN apt update && \
     apt install -y libvirt-dev && \
-    pip3 install sushy-tools==0.18.1 libvirt-python
+    apt clean && \
+    pip3 install --no-cache-dir sushy-tools==${SUSHY_TOOLS_VERSION} libvirt-python
+
+ENV SUSHY_EMULATOR_CONFIG="/root/sushy/conf.py"
+ENV SUSHY_EMULATOR_LISTEN_IP="::"
+ENV SUSHY_EMULATOR_LISTEN_PORT="8000"
+ENV SUSHY_EXTRA_FLAGS="--debug"
```

Comment:
Should be `SUSHY_EMULATOR_EXTRA_FLAGS` for consistency

Reply:
To be fair, that's not a sushy-emulator env variable, that's why I choose to _rename_ it:

```
sushy-emulator -h
usage: sushy-emulator [-h] [--config CONFIG] [--debug] [-i INTERFACE] [-p PORT] [--ssl-certificate SSL_CERTIFICATE]
                      [--ssl-key SSL_KEY] [--os-cloud OS_CLOUD | --libvirt-uri LIBVIRT_URI]

optional arguments:
  -h, --help            show this help message and exit
  --config CONFIG       Config file path. Can also be set via environment variable SUSHY_EMULATOR_CONFIG.
  --debug               Enables debug mode when running sushy-emulator.
  -i INTERFACE, --interface INTERFACE
                        IP address of the local interface to listen at. Can also be set via config variable
                        SUSHY_EMULATOR_LISTEN_IP. Default is all local interfaces.
  -p PORT, --port PORT  TCP port to bind the server to. Can also be set via config variable SUSHY_EMULATOR_LISTEN_PORT. Default is
                        8000.
  --ssl-certificate SSL_CERTIFICATE
                        SSL certificate to use for HTTPS. Can also be set via config variable SUSHY_EMULATOR_SSL_CERT.
  --ssl-key SSL_KEY     SSL key to use for HTTPS. Can also be setvia config variable SUSHY_EMULATOR_SSL_KEY.
  --os-cloud OS_CLOUD   OpenStack cloud name. Can also be set via environment variable OS_CLOUD or config variable
                        SUSHY_EMULATOR_OS_CLOUD.
  --libvirt-uri LIBVIRT_URI
                        The libvirt URI. Can also be set via environment variable SUSHY_EMULATOR_LIBVIRT_URI. Default is
                        qemu:///system
```

But I can modify it `SUSHY_EMULATOR_EXTRA_FLAGS`, sure